### ISSUE TITLE
Closed loop slew functionality 

### ISSUE THREAD

**Issue by Physarah** [wtgee/huntsman-pocs#536]
This is just a wish list item for me. Enabling closed loop slew before we start an observation would ensure we are always on target.

**Comment by danjampro** [comment]
@Physarah What is the main barrier in getting this done? The interface between `POCS` and TSX?

**Comment by Physarah** [comment]
I believe so. The trigger from POCS to initiate the closed loop slew (with guide cam presumably), then ensuring POCS doesn't move the mount again afterwards.

**Comment by danjampro** [comment]
Am I correct in thinking this is this something that is already implemented in TSX GUI?

**Comment by Physarah** [comment]
yup - so it's just a matter of triggering this from POCS some how

**Comment by danjampro** [comment]
I think it would require a new item in `POCS` here:

https://github.com/panoptes/POCS/tree/develop/resources/bisque/mount

The only problem is that I have no idea how to write these things. I think @wtgee wrote them originally.

@wtgee, do you have any pointers on how to do this?
